packages/components/pie-cookie-banner/src/localisation-utils.ts

```diff
@@ -0,0 +1,137 @@
+import {
+    TemplateResult,
+} from 'lit';
+
+import {
+    type CookieBannerLocale,
+    type CustomTagEnhancers,
+} from './defs';
+
+function logError (message:string):void {
+    console.error(`[localisation-utils]: ${message}`);
+}
+
+function getObjectKeyValue (obj:object, path:string):string {
+    // Implementation based on You-Dont-Need-Lodash-Underscore:
+    // https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_get
+    const travel = (regexp:RegExp) => String.prototype.split
+        .call(path, regexp)
+        .filter(Boolean)
+        .reduce((acc:object, key:string) => {
+            if (acc !== null && acc !== undefined) {
+                // TODO: Provide proper TS solution for this
+                // @ts-expect-error - Object is dynamic
+                return acc[key];
```

Comment:
TS magic required: how to solve this dynamically?

Reply:
@fernandofranca I think if you swap `object` for `Record<string, any>` on lines 14 and 20 that should help